app/models/user.rb

```diff
@@ -33,4 +33,8 @@ def full_name
   def support_user?
     false
   end
+
+  def service
+    self.class.name.split("::").first.downcase.to_sym
+  end
```

Comment:
A comment could be nice here, because this line seems quite difficult to parse:

```suggestion
  def service
    # Infer service from class namespace – e.g. Placements::User becomes :placements
    self.class.name.split("::").first.downcase.to_sym
  end
```

Reply:
Updated. Added examples and method documentation using standard RDoc/YARD documentation format.